### ISSUE TITLE
Which parameters for (uiop:quit CODE FINISH-OUTPUT)?

### ISSUE THREAD

**Issue by Ambrevar** [atlas-engineer/nyxt#2368]
In commit 0b1f8244d02d64322cdc45dd8b64ecb217bcbaff @jmercouris added this line to quit Nyxt:

```lisp
(uiop:quit 0 nil)
```

On SBCL `uiop:quit` calls to `sb-ext:exit`.  Excerpt from the docstring:

```
    When ABORT is false (the default), current thread is first unwound,
    *EXIT-HOOKS* are run, other threads are terminated, and standard
    output streams are flushed before SBCL calls exit(3) -- at which point
    atexit(3) functions will run. If multiple threads call EXIT with ABORT
    being false, the first one to call it will complete the protocol.
```

So the different between T and NIL in `uiop:quit` here is whether `sb-ext:*exit-hooks*` are run or not, respectively.

Thoughts?

**Comment by CorruptedVor** [comment]
Looks like the hooks should run to make sure there is no buffered data lost, like a write to a file somewhere.
TL;DR clean exit, `(uiop:quit 0 nil)` being the proper way?

**Comment by jmercouris** [comment]
Well then, seems the code is correct :-)
